src/audio/chain_dma.c

```diff
@@ -303,7 +303,6 @@ static int chain_task_start(struct comp_dev *dev)
 	}
 
 	pm_policy_state_lock_get(PM_STATE_RUNTIME_IDLE, PM_ALL_SUBSTATES);
-	cd->chain_task.state = SOF_TASK_STATE_INIT;
```

Comment:
Can we also remove 'schedule_task_cancel(&cd->chain_task)' in 'chain_task_pause' ? I think after this change it will not be necessary.

Reply:
@makarukp this should be a separate PR, reverting that change - after this is merged